packages/suite/src/components/firmware/FirmwareInitial.tsx

```diff
@@ -89,11 +97,11 @@ const FirmwareInitial = ({ cachedDevice, setCachedDevice, standaloneFwUpdate }:
             heading: <Translation id="TR_INSTALL_FIRMWARE" />,
             description: (
                 <Translation
-                    id={getDescription(
-                        device.firmware === 'required',
-                        !!standaloneFwUpdate,
-                        !!device.firmwareRelease?.isLatest,
-                    )}
+                    id={getDescription({
+                        required: device.firmware === 'required',
+                        standaloneFwUpdate,
+                        reinstall: device.firmware === 'valid',
```

Comment:
Could we add a comment explaning this in similar way as you did [here](https://github.com/trezor/trezor-suite/pull/3961#issuecomment-871417500)?

Reply:
done, thanks